lib/print.js

```diff
@@ -62,7 +62,21 @@ const printApply = (node, parent) => {
         }
         return result;
     } else if (op === 'pow') {
-        return `${print(args[0], node)}^${print(args[1], node)}`;
+        const exponent = node.args[1];
+        let bool = false;
+        // check if exponent has multiple terms and if the exponent
+        // contains a div/mul operation
+        // e.g x^((x + 1)/(2 + 2))
+        if (exponent.args){
+            if (exponent.op == 'div' || exponent.op == 'mul'){
```

Comment:
There should be a space between conditionals the opening brace.  See https://github.com/semantic-math/math-parser/issues/37.  It can't be handled in a separate diff.